themes/default/content/testimonials/_index.md

```diff
@@ -165,4 +165,18 @@ messages:
       link: https://pulumi.slack.com/archives/C4PCU9XKN/p1683131107652869
       source: email
       featured: true
+    -
+      message: "I have been working on AWS for quite some time. But I never used IaC.
+      Pulumi is my first IaC framework and I am loving it. I have never been a fan of Cloudformation and its UI confuses me. Will be using Pulumi quite often."
```

Comment:
CloudFormation

also, can we improve the grammar a bit?  (collapse the second sentence into 1st)  ... I would also kill last two sentences; always take the high road when you can

Reply:
i usually put these quotes in verbatim for authenticity. here is the edit: https://github.com/pulumi/pulumi-hugo/pull/3243